irods/exception.py

```diff
@@ -5,6 +5,8 @@
 from __future__ import absolute_import
 import six
 import numbers
+import errno
+import os
```

Comment:
alphabetize?

Reply:
These still appear to not be alphabetized (at least in relation to the rest of the imports).

Reply:
still not alphabetized